tests/opencode/test-plugin-bootstrap-behavior.sh

```
@@ -0,0 +1,105 @@
+
+PLUGIN_FILE="$HOME/.config/opencode/superpowers/.opencode/plugins/superpowers.js"
+USING_SKILL_PATH="$HOME/.config/opencode/superpowers/skills/using-superpowers/SKILL.md"
+
+if [ ! -f "$PLUGIN_FILE" ]; then
+    echo "  [FAIL] Plugin file not found at $PLUGIN_FILE"
+    exit 1
+fi
+
+# Helper: run a small Node snippet that exercises experimental.chat.system.transform
+run_transform() {
+    local expect_diagnostic="$1"
```

Shellcheck (0.11.0)

[warning] expect_diagnostic appears unused. Verify use (or export if used externally).

(SC2034)